sketch.js

```diff
@@ -1,13 +1,44 @@
 let r, g, b;
 let database;
-let rgbDiv;
 
 function pickColor() {
-  r = floor(random(256));
-  g = floor(random(256));
-  b = floor(random(256));
-  background(r, g, b);
-  rgbDiv.html(`R:${r} G:${g} B:${b}`);
```

Comment:
I put this in here so that anyone using a screen reader would be able to hear the RGB numbers? But perhaps there is a better way to make it accessible.

Reply:
Honestly, I hadn't thought of that. I just know that I found myself "cheating" a lot when I wasn't confident about my choice.
I'm not sure what they are, but I know there are some tag properties that are explicitly intended for screen readers and could be used instead. As a workaround, I'm assuming that the text color can just be made the same as the background color.